app/main/views/services.py

```diff
@@ -41,6 +43,10 @@ def list_services():
 
     services = Service.query.filter(
         Service.framework.has(Framework.expired == false())
+    ).order_by(
```

Comment:
I'm not sure what you mean by "JSON isn't supposed to be ordered". We return a list of services, lists are inherently ordered.

In case of postgres, not having ordered_by results in the "whatever is fastest" order and it's not really clear what are the guarantees it provides: i.e. does it stay the same between page requests, or is it possible to get duplicate services if order changes between page requests.